lib/ads/ad_manager.js

```diff
@@ -57,7 +57,7 @@ shaka.ads.AdManager = class extends shaka.util.FakeEventTarget {
    * @export
    */
   initClientSide(adContainer, video) {
-    if (!window['google'] || !google.ima) {
+    if (!window['google'] || !google.ima || !google.ima.AdsRequest) {
```

Comment:
Please add a comment about the AdsRequest part testing for the CS SDK specifically (vs SS)

Reply:
I added the comment